shared/AppInsightsCommon/src/TelemetryItemCreator.ts

```diff
@@ -35,6 +35,11 @@ export function createTelemetryItem<T>(item: T,
     if (item[strIkey]) {
         iKey = item[strIkey];
         delete item[strIkey];
+        //TODO: add connectionString Config
+        var _console = typeof console !== strUndefined ? console : null;
```

Comment:
Same comment as above -- this will always occur as it will always be populated -- even when using a connection string.

Reply:
so it is in the `createTelemetryItem` function, I thought `ikey` here should be overwritten ikey instead of the one from configuration.
```
  let iKey = "";
    if (item[strIkey]) {
        iKey = item[strIkey];
        delete item[strIkey];
        //TODO: add connectionString Config
        var _console = typeof console !== strUndefined ? console : null;
        if (_console && _console.warn) {
            _console.warn("Instrumentation key based ingestion will be no longer supported.");
        }
    }

```